src/components/modal/index.tsx

```diff
@@ -80,7 +80,7 @@ class Modal extends Component<ModalProps> {
   renderTouchableOverlay() {
     const {testID, overlayBackgroundColor, onBackgroundPress, accessibilityLabel = 'Dismiss'} = this.props;
     if (_.isFunction(onBackgroundPress) || !!overlayBackgroundColor) {
-      const isScreenReaderEnabled = Constants.accessibility.isScreenReaderEnabled;
+      const isScreenReaderEnabled = !Constants.isWeb ? Constants.accessibility.isScreenReaderEnabled : false;
```

Comment:
Consider changing this expression to 
```suggestion
      const isScreenReaderEnabled = !Constants.isWeb && Constants.accessibility.isScreenReaderEnabled;
```